packages/react-core/src/components/Tooltip/examples/Tooltip.md

```diff
@@ -61,63 +48,159 @@ OnIconTooltip = () => (
 )
 ```
 
-```js title=Positions
+```js title=Options
 import React from 'react';
-import { Button, Tooltip, TooltipPosition, Checkbox } from '@patternfly/react-core';
-
-class TooltipPositions extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      position: TooltipPosition.top,
-      keepInViewChecked: true
-    };
-    this.handleKeepInViewChange = checked => {
-      this.setState({ keepInViewChecked: checked });
-    };
-  }
+import { Button, Tooltip, Checkbox, Select, SelectOption, TextInput } from '@patternfly/react-core';
 
-  render() {
-    return (
+OptionsTooltip = () => {
+  const [trigger, setTrigger] = React.useState(['mouseenter', 'focus']);
+  const [isVisible, setIsVisible] = React.useState(true);
+  const [contentLeftAligned, setContentLeftAligned] = React.useState(false);
+  const [enableFlip, setEnableFlip] = React.useState(true);
+  const [position, setPosition] = React.useState('top');
+  const [positionSelectOpen, setPositionSelectOpen] = React.useState(false);
+  const [flipSelectOpen, setFlipSelectOpen] = React.useState(false);
+  const [flipBehavior, setFlipBehavior] = React.useState('flip');
+  const [entryDelayInput, setEntryDelayInput] = React.useState(0);
+  const [exitDelayInput, setExitDelayInput] = React.useState(0);
+  const [animationDuration, setAnimationDuration] = React.useState(300);
+  const tipBoxRef = React.useRef(null);
+  const scrollToRef = ref => ref && ref.current && (ref.current.scrollTop = 400);
+  React.useEffect(() => {
+    scrollToRef(tipBoxRef);
+  }, []);
+  return (
+    <>
       <div>
-        <div>
-          <span style={{ paddingRight: '10px' }}>Tooltip Position</span>
-          <select
-            aria-label="Tooltip position"
-            onChange={event => {
-              this.setState({ position: event.target.value });
+        <div style={{ border: '1px solid'}}>
```

Comment:
Can you remove this border?

Reply:
I could but I like it :P, any particular reason?

Reply:
It doesn't really match the rest of our examples. Our layouts used a dashed border for example, and our forms don't have a border.

Reply:
In this case it makes it more obvious that the text is an interactive element which is why I added it